packages/server/src/websockets/index.ts

```diff
@@ -3,15 +3,19 @@ import Koa from "koa"
 import ClientAppSocket from "./client"
 import GridSocket from "./grid"
 import BuilderSocket from "./builder"
+import env from "../environment"
 
-let clientAppSocket: ClientAppSocket
-let gridSocket: GridSocket
-let builderSocket: BuilderSocket
+let clientAppSocket: ClientAppSocket | undefined
+let gridSocket: GridSocket | undefined
+let builderSocket: BuilderSocket | undefined
 
 export const initialise = (app: Koa, server: http.Server) => {
-  clientAppSocket = new ClientAppSocket(app, server)
-  gridSocket = new GridSocket(app, server)
-  builderSocket = new BuilderSocket(app, server)
+  // currently we use ioredis-mock which doesn't work with the websocket system for testing
+  if (!env.isTest()) {
```

Comment:
Is this still needed even with ioredis-mock removed now? Just since the comment mentions it.

Reply:
It is not - good catch!